packages/circuit-ui/components/TopNavigation/TopNavigation.stories.tsx

```diff
@@ -131,6 +133,7 @@ export const WithSideNavigation = (args: TopNavigationProps) => {
         <SideNavigation
           {...sideNavigationProps}
           isOpen={isSideNavigationOpen}
+          skipNavigationHref={undefined}
```

Comment:
Why are you setting a value above and overriding it to `undefined` here?

Reply:
the `sideNavigationProps` already has a `skipNavigationHref` and I want to avoid having to skip links in the same page 😄

Reply:
Ah, right, I missed that this is the story with both the top and side navigation 🤦🏻